src/Anthropic/Model/Claude.php

```diff
@@ -15,17 +15,18 @@ final class Claude implements LanguageModel
 {
     public function __construct(
         private ClaudeRuntime $runtime,
-        private Version $version = Version::SONNET_35,
+        private ?Model $model = null,
```

Comment:
Not sure about the name `Model`, because now we have `Model` and `LanguageModel` 🤔 

Or we keep `Version` and remove the enums.

Reply:
Maybe `ModelVersion`?

Reply:
yes, how about `ModelVersion` and inlining the enums as constants that are used for named constructor?

like `ModelVersion::sonnet35()` or `ModelVersion::gpt4o()`

WDYT?

Reply:
Will rework this PR to get a feeling